### ISSUE TITLE
JetWooBuilder - problem with Custom Archive Category - when custom shop is ON, categories cannot be open

### ISSUE THREAD

**Issue by poziomapl** [CrocoBlock/suggestions#2095]
**Describe the bug**
When I enable Custom Shop Page and I choose my shop template from the list it works for shop - like it should be. But it causes a problem to enter the Product Category Archive. When you click on a Product Category, you are redirected to the Custom Shop page.

This is despite the fact that I have set a Custom Archive Template for a Product Category to enabled. What's more, when the field is unchecked (then the default woocommerce template or elementor works), you cannot enter the archive - even when this field is not active and it should work by default with woocommerce template page.

Only when I turn off the Custom Shop Page - you can enter into the product categories.

And when the Custom Archive Category is turned on and set, unfortunately it also doesn't work and nothing happens as if this feature is broken

**To Reproduce**
1. Set Enable custom shop page to on.
2. Try enter into the Procuct Category Archive - it does not work.
3. Set Enable custom shop page to off - and product cat. archive will work.

or 

1. Set Custom Archive Category to on and select form list - it does not work as above.

**Plugin Version**
JetWooBuilder 1.7.1

**Comment by Floydtm** [comment]
@poziomapl, hi! Could you please contact our [support](https://crocoblock.com/help-center/) team and provide access to your site. The developer will check everything and fix it. Thank you!

**Comment by Emoncht** [comment]
Have you solved the problem? i am facing the same error with martfury theme

**Comment by gbujalil** [comment]
Hi! I have the same issue. Could you post the solution to this?

**Comment by kitsunecreativeco** [comment]
I have this same issue too

**Comment by JunLargo** [comment]
I have the same issue but I found a solution. I disable the custom shop page in Jetwoobuilder under WooCommerce settings but leave the custom archive category to jetwoobuilder template . I created a shop template page using Elementor theme builder but still using the jetwoobuilder widgets and set the condition to shop page. So my shop page is using elementor template and my category template is pointing to jetwoobuilder category template page.
